app/controllers/concerns/course/forum/topic/subscribe_concern.rb

```diff
@@ -0,0 +1,25 @@
+module Course::Forum::Topic::SubscribeConcern
+  extend ActiveSupport::Concern
```

Comment:
Same thing. Tell you what, I'll review this one more time. I am going to *close* this, and please split it up.